src/cpp/benders/benders_core/BendersBase.cpp

```diff
@@ -632,7 +627,9 @@ Output::SolutionData BendersBase::BendersSolution() const {
         best_iteration_data.invest_cost,
         best_iteration_data.subproblem_cost,
         best_iteration_data.invest_cost + best_iteration_data.subproblem_cost,
-        candidates_vec};
+        candidates_vec,
+        0
+    };
```

Comment:
I'm not sure I understand why this 0 is required here ?

Reply:
This is the value of the 12th member : cumulative_number_of_subproblem_resolved. Better to specify all values